Marble Blast Platinum/platinum/client/scripts/optionsGui.cs

```diff
@@ -445,12 +445,15 @@
 $Options::Name    ["Online", 5] = "globalSize";
 $Options::Title   ["Online", 5] = "Global Score Page Size";
 $Options::Type    ["Online", 5] = "value";
-$Options::Name    ["Online", 6] = "autoLogin";
-$Options::Title   ["Online", 6] = "Auto Login";
-$Options::Type    ["Online", 6] = "boolean";
+$Options::Name    ["Online", 6] = "chatMessageSize";
+$Options::Title   ["Online", 6] = "Ingame Lines of Chat";
```

Comment:
There needs to be a hyphen here, it should be "In-Game Lines of Chat"

Reply:
In my opinion "Ingame" looked better, but sure.